test_segmentation_model.py

```diff
@@ -50,7 +50,8 @@ def test(gcp_bucket, dataset_id, model_id, batch_size):
         Path(local_dataset_dir, dataset_id, 'test').as_posix(),
         rescale=1./255,
         target_size=target_size,
-        batch_size=batch_size)
+        batch_size=batch_size,
+        seed=None)  # train_config['test_data_shuffle_seed'])
```

Comment:
same comment as in train -- we should check to see if it's set and use it, otherwise use None

Reply:
done